config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
 
+  resources :gems_adoptions
```

Comment:
If using the `resources` route helper we should limit it to the routes we're actually interested in exposing, so this should probably look like: `resources :gems_adoptions, only: [:create, :destroy]`.